nautobot/dcim/models/device_components.py

```diff
@@ -265,6 +266,8 @@ def parent(self):
 
 @extras_features(
     "cable_terminations",
+    "custom_fields",
```

Comment:
Shouldn't need to add this - `custom_fields` is implicit on all models that support them for the past few releases.

Reply:
removed it.